files/metainfo.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.clashcross.clashcross</id>
+
+  <name>ClashCross</name>
+  <summary>ClashCross is a debugging application built on the Clash core, supporting Linux.</summary>
```

Comment:
```suggestion
  <summary>ClashCross is a debugging application built on the Clash core</summary>
```

But given that it doesn't say what "clash core" is, it's still to cryptic for a summary.